docs/installation/hetzner-cloud.md

```diff
@@ -0,0 +1,129 @@
+```
+cd /home
+git clone https://github.com/fossasia/yaydoc
+```
+
+## Deploy the application
+After cloning lets navigate to the folder.
+```
+cd /home/yaydoc
+```
+Now to setup environment variables
+```
+cp .env.example .env
+```
+Copy and change all the fields within .env accordingly. To open it in an editor type:
+```
+nano .env
+```
+Install all of the dependencies
+```
+npm install
+pip install -r requirements.txt
+```
+Now deploy the application using the following command. 
+```
+node ./bin/www
+```
+You can view it at `localhost:3001` or in this case `<server-ip>:3001`.
+
+But currently it will stop running when your ssh connection is closed.
+
+## Complete Deployment
+This is so it will be running forever while on the server.
+
+We will be using a tool known as `cron` to do this.
+
+After installing cron we can type the following command to open crontab to schedule a command.
+```
+crontab -e
+```
+Using `vi` commands you can add the following line to the end to create a new cronjob.
+```
+* * * * * root node ./home/yaydoc/bin/www
+```
+What this does is schedule the `node ./home/yaydoc/bin/www` command by user `root` every minute. You can read more about cron [here](https://www.taniarascia.com/setting-up-a-basic-cron-job-in-linux/) if you would like to.
+
+Now going back to `<server-ip>:3001` we should see it running properly!
+
+# Setup continuous integration.
+What if we wanted to update the deployment everytime there is new code pushed to the repository?
+
+We can do that with the help of a simple [script](https://github.com/logsol/Github-Auto-Deploy).
+
+Let's set it up.
+```
+cd /home
+git clone https://github.com/logsol/Github-Auto-Deploy.git
+```
+Now 
+```
+cd Github-Auto-Deploy
+```
+Copy 	GitAutoDeploy.conf.json.example and change it accordingly.
+```
+cp GitAutoDeploy.conf.json.example GitAutoDeploy.conf.json
+nano GitAutoDeploy.conf.json
+```
+Change the file to the following
+```
+{
+	"port": 3001, 
+	"repositories": 
+	[{
+		"url": "https://github.com/fossasia/yaydoc", 
+		"path": "/home/yaydoc/",
+		"deploy": "npm install ; pip install -r requirements.txt ; PORT=8000 node ./bin/www"
+	}]
+}
+```
+Make your changes and use `CTRL + X` to save and return.
+
+This uses port 3001 to listen to any changes in the GitHub repository and deploys the application on port 8000.
+
+Make sure to add a webhook pointing to your server ip at port 3001 in the GitHub repository.
+
+Now we must update the cron job so instead of running the node application, it runs this script instead.
+
+Open crontab using
+```
+crontab -e
+```
+
+Now change the last line to
+```
+* * * * * sudo python /home/Github-Auto-Deploy/GitAutoDeploy.py --daemon-mode
+```
+Now it should be running successfully. Try going to `<server-ip>:3001` and you will be left with a GET error because it only accepts POST requests.
+
+After the next commit/merge to the repository, the app should successfully be running on `<server-ip>:8000`
```

Comment:
Please add info how to make it run on custom domain and provide links to further documentation.